index.html

```diff
@@ -1,13 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="styles/index.css">
-</head>
-<body>
-  <h1>Quiz-App</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App start page</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body class="app-grid">
+    <header id="#starthome" class="top">
+      <h1 class="top quiz-app-heading">Quiz-App</h1>
+    </header>
+
+    <main class="app-grid__scrollarea h-100vh">
+      <section class="quiz-app-card">
+        <article class="quiz-app-card-question">
+          <h2 class="quiz-app-card__heading">Question:</h2>
+          <p class="quiz-app-card-question--text"></p>
+        </article>
+        <figure class="bookmark-container">
+          <img src="" alt="" width="" height="" class="bookmark--on" />
+        </figure>
+        <button class="quiz-app-card-answer--show"></button>
+        <button class="quiz-app-card-answer--hide"></button>
+        <p class="quiz-app-card-question--answer"></p>
+        <div class="quiz-app-card-advise-tag__container">
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+        </div>
+      </section>
+    </main>
+
+    <nav class="nav-container">
+      <ul class="nav-container__list">
+        <li class="nav-container__listitem">
+          <a href="#starthome"
+            ><img
+              src=""
+              width=""
+              height=""
+              alt=""
+              class="nav-container__home-icon"
```

Comment:
the naming again 😺 i would recommend to name all of your `img` tags for eg `class="nav-container__image"` or `class="nav-container__icons" `
you can do this the same way to the other tags in this part like the `a`, `li` tags
for eg all `a` tags `class="nav-container__links"`
the reason is...later you can style them all with just one class name ;)

Reply:
yes you are right - my "bem thought - lol" to share with you, was, to add an utility-class for the styling of the mini-container, each of the 4 are sitting in and getting their position - what do you think?